src/Unit/Unit.sqf

```diff
@@ -2666,12 +2667,23 @@ CLASS("Unit", ["Storable" ARG "GOAP_Agent"])
 	ENDMETHOD;
 
 	public override METHOD(deserializeFromStorage)
-		params [P_THISOBJECT, P_ARRAY("_serial")];
+		params [P_THISOBJECT, P_ARRAY("_serial"), P_NUMBER("_version")];
 		_serial set [UNIT_DATA_ID_OWNER, 2]; // Server
 		_serial set [UNIT_DATA_ID_MUTEX, MUTEX_NEW()];
 		_serial set [UNIT_DATA_ID_OBJECT_HANDLE, objNull];
 		_serial set [UNIT_DATA_ID_AI, ""];
 
+		// SAVEBREAK >>>
```

Comment:
@SomethingSimpl you can see how the versioning works by this example. saveVersion.hpp holds the current save game version value. It was 28, now I set it to 29 when I we add this feature. Here I check if the version of the loaded game is lower than 29, and if so, correct the data upon loading.